include/cadmium/engine/common_helpers.hpp

```diff
@@ -40,6 +40,23 @@ namespace cadmium {
             std::apply(for_each_fold_expression, ts);
         }
 
+        std::string join(std::vector<std::string> v) {
```

Comment:
In other places I called this "implode" based in other languages "implode/explode" functions.
For example here: ./include/cadmium/logger/common_loggers_helpers.hpp +71

Reply:
Ok, I am more used to the python / javascript styles. 

In order to not use the same name as in the common_logger, do you want me to rename this method as "implode_vector" or you prefer to call it "implode"?